js/viewmodel.js

```diff
@@ -65,7 +138,7 @@ window.FBE = window.FBE || {};
 
     function addDerivedProperties(entity) {
 
-        var nonRotators = /beacon|roboport|lab|heat_pipe|pipe|furnace|chest|pole|substation|solar|accumulator|centrifuge|wall|rocket_silo|radar|turret|speaker|power_switch/i;
+        var nonRotators = /beacon|roboport|lab|heat_pipe|pipe|furnace|chest|pole|substation|solar|accumulator|centrifuge|wall|rocket_silo|radar|turret|speaker|power_switch|reactor|lamp|land_mine/i;

```

Comment:
well, that be a piece of regex...

wouldn't an array and `#indexOf` work just as well?